Makefile

```diff
@@ -342,8 +342,8 @@ clean-test: ## clean up debris from previously failed tests
 
 install: ## install binaries
 	@echo "$(WHALE) $@ $(BINARIES)"
-	@mkdir -p $(DESTDIR)/bin
-	@install $(BINARIES) $(DESTDIR)/bin
+	@mkdir -p $(DESTDIR)/usr/bin
+	@install $(BINARIES) $(DESTDIR)/usr/bin
```

Comment:
DESTDIR is set to `/usr/local`, so you are attempting to install the binaries to `/usr/local/usr/bin`
https://github.com/containerd/containerd/blob/034dbd07c3a38129519613852acea1577190c836/Makefile#L20

Reply:
Ah, okay, the DESTDIR setting is wrong. I'll send another patch ...